unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/Tests/FriendsControllerShould.cs

```diff
@@ -68,14 +69,60 @@ public void AddFriendRequests()
             };
             controller.OnUpdateFriendship += (s, action) => friendsUpdated[s] = action;
 
-            apiBridge.OnFriendRequestsAdded += Raise.Event<Action<AddFriendRequestsPayload>>(
+            apiBridge.GetFriendRequests(0, 0, 0, 0).Returns(info => UniTask.FromResult(
```

Comment:
i think you dont need the lambda with `info` parameter, you could just do: `Returns(UniTask.FromResult(..))`

Reply:
True! Fixed!